cmd/sqlpipe/system_mysql.go

```diff
@@ -742,7 +742,16 @@ func (system Mysql) getSqlFormatters() (
 
 func (system Mysql) getTableColumnInfosRows(schema, table string) (rows *sql.Rows, err error) {
 	query := fmt.Sprintf(`
-		WITH PrimaryKeys AS (
+		SELECT
```

Comment:
Missing         SELECT DISTINCT

Reply:
When the column is grouped in the subquery it doesn't need distinct.

Reply:
I ran into an issue in the results where that DISTINCT was needed.   I guess you can try it out and see for yourself?

Reply:
You are correct with using of the grouping.  Good call.

Reply:
Thank you for the attentive review.